Bucketeer/Sources/Internal/Evaluation/EvaluationInteractor.swift

```diff
@@ -120,4 +139,52 @@ final class EvaluationInteractorImpl: EvaluationInteractor {
     func clearUpdateListeners() {
         updateListeners.removeAll()
     }
+
+    private func updateFeatureTag(value: String) {
+        // https://github.com/bucketeer-io/android-client-sdk/issues/69
+        // 2- Save the featureTag in the UserDefault if it is configured in the BKTConfig
+        // 3- Clear the userEvaluationsID in the UserDefault if the featureTag changes
```

Comment:
Can you update the number?
Also, please keep the numbered-list format. Currently, we have `1- ` and `1. `

Reply:
fixed